src/WaterPokemon.java

```diff
@@ -0,0 +1,65 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class WaterPokemon extends Pokemon {
+    public List<String> attacks = Arrays.asList("surf", "hydroPump", "hydroCannon", "rainDance");
+
+    public WaterPokemon(String name, int level, int hp, String food, String sound){
+        super(name, level, hp, food, sound, "water");
+    }
+
+    public List<String> getAttacks(){
+        return attacks;
+    }
+
+    public void surf(Pokemon name, Pokemon enemy){
+        System.out.println(name.getName() + " attacks " + enemy.getName() + " with surf");
+        double damage = damage(enemy, 15);
+        System.out.println(enemy.getName() + " loses " + damage + "hp");
+        enemy.setHp(-damage);
+        System.out.println(enemy.getName() + " Has " + enemy.getHp() + "hp left");
+    }
+
+    void hydroPump(Pokemon name, Pokemon enemy){
+        System.out.println(name.getName() + " attacks " + enemy.getName() + " with hydro pump");
+        double damage = damage(enemy, 23);
+        System.out.println(enemy.getName() + " loses " + damage + "hp");
+        enemy.setHp(-damage);
+        System.out.println(enemy.getName() + " Has " + enemy.getHp() + "hp left");
+    }
+    void hydroCanon(Pokemon name, Pokemon enemy){
+        System.out.println(name.getName() + " attacks " + enemy.getName() + " with hydro canon");
```

Comment:
Herhaling van wat je hierboven hebt getypt. misschien had dit wel in een methode gekund ;)